ui/tests/acceptance/enterprise-namespaces-test.js

```diff
@@ -28,51 +38,83 @@ module('Acceptance | Enterprise | namespaces', function (hooks) {
     fetchSpy.restore();
   });
 
+  test('it focuses the search input field when the component is loaded', async function (assert) {
+    assert.expect(1);
```

Comment:
This is probably a good thing to add to our rules document, but you really only need `assert.expect` if there's a chance the test could run without all the assertions being hit. This is common when using mirage when you have an assertion inside a this.server.x because if that network request doesn't get hit neither will the assertion and the test could still pass. 

In the cases here it appears that all the assertions will be hit, and so we don't need to have the .expect catch. 

let me know if that does/doesn't make sense.

Reply:
ah okay that's good to know - I thought it as a fail-safe pattern because I've seen it a lot, but that makes sense!